helpers/CategoryList.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace app\helpers;
+
+use yii\helpers\Url;
+
+class CategoryList
+{
+    public static function generate($items)
+    {
+        $html = '';
+
+        $length = count($items);
+
+        for ($i = 0; $i < $length; $i++) {
+            $point = $i != $length - 1 ? ',&nbsp;' : '';
+
+            $html .= '<a href="' . Url::to($items[$i]['url']) . '">' . $items[$i]['label'] . $point . '</a>';
```

Comment:
Don't use plain html here. All the html should be in templates. You can use Html::tag helper to render any tag you need with attributes specified https://github.com/yiisoft/yii2/blob/master/docs/guide/helper-html.md
Also you can write your own helper it will be at the next lesson